modules/spark/src/test/scala/org/apache/ignite/spark/IgniteSQLDataFrameSpec.scala

```diff
@@ -270,6 +271,44 @@ class IgniteSQLDataFrameSpec extends AbstractDataFrameSpec {
             persons(0).getAs[Long]("city_id") should equal(2)
             persons(0).getAs[Long]("count(1)") should equal(3)
         }
+
+        it("should use the schema name where one is specified") {
+            // `employeeCache1` is created in the schema matching the name of the cache, ie. `employeeCache1`.
+            createEmployeeCache(client, "employeeCache1")
+
+            spark.read
+                .format(FORMAT_IGNITE)
+                .option(OPTION_CONFIG_FILE, TEST_CONFIG_FILE)
+                .option(OPTION_TABLE, "employee")
+                .option(OPTION_SCHEMA, "employeeCache1")
+                .load()
+                .createOrReplaceTempView("employeeWithSchema")
+
+            // `employeeCache2` is created with a custom schema of `employeeSchema`.
+            createEmployeeCache(client, "employeeCache2", Some("employeeSchema"))
+
+            // Remove a value from `employeeCache2` so that we know whether the select statement picks up the
+            // correct cache, ie. it should now have 2 values compared to 3 in `employeeCache1`.
+            Iterators.size(client.cache("employeeCache2").iterator()) should equal(3)
+            val removed = client.cache("employeeCache2").remove("key1")
+            removed shouldBe true
```

Comment:
Can we just make it
`client.cache("employeeCache2").remove("key1") shouldBe true`?